src/plugins/InspectorPlugin.tsx

```diff
@@ -23,9 +23,9 @@ const InspectorPlugin: React.FC<IMethodPluginProps> = (props) => {
         openrpcMethodObject: props.openrpcMethodObject,
         request: {
           jsonrpc: "2.0",
-          id: 0,
           method: method.name,
           params: exampleParams || [],
+          id: 0,
```

Comment:
aesthetically more pleasing?

Reply:
![image](https://user-images.githubusercontent.com/364566/78091438-a554e080-7381-11ea-9622-510ac129c870.png)

inspector does this order for its defaults

Reply:
👍